qutip/tests/core/test_qobjevo.py

```diff
@@ -176,42 +175,43 @@ def _rand_t(self):
     def test_param(self, tlist):
         assert tlist is self.tlist
 
-
-    @pytest.mark.parametrize(['form', 'base', 'kwargs'],
-        [pytest.param('cte', cte_qobj, {}, id='cte'),
-         pytest.param('func', [cte_qobj,
-                               [cplx_qobj, cplx_coeff.func]],
-                      {'args':args}, id='func'),
-         pytest.param('string', [cte_qobj,
-                                 [cplx_qobj, cplx_coeff.string]],
-                      {'args':args}, id='string'),
-         pytest.param('spline', [[cplx_qobj, cplx_coeff.spline(tlist, args)],
-                                 cte_qobj],
-                      {}, id='spline'),
-         pytest.param('array', [cte_qobj,
-                                [cplx_qobj, cplx_coeff.array(tlist, args)]],
-                      {'tlist':tlist}, id='array'),
-         pytest.param('array_log', [cte_qobj,
-                                [cplx_qobj, cplx_coeff.array(tlistlog, args)]],
-                      {'tlist':tlistlog}, id='array_log'),
-         pytest.param('with_args', [cte_qobj,
-                                    [cplx_qobj, cplx_coeff.func],
-                                    [real_qobj, real_coeff.string]],
-                      {'args':args}, id='with_args'),
-         pytest.param('no_args', [cte_qobj,
-                                  [cplx_qobj, cplx_coeff.spline(tlist, args)],
-                                  [real_qobj, real_coeff.array(tlist, args)]],
-                      {'tlist':tlist}, id='no_args'),
-         pytest.param('mixed_tlist', [cte_qobj,
-                        [cplx_qobj, coefficient(
-                                        cplx_coeff.array(tlist, args),
-                                        tlist=tlist
-                                    )],
-                        [real_qobj, coefficient(
-                                        real_coeff.array(tlistlog, args),
-                                        tlist=tlistlog
-                                    )]],
-                      {}, id='mixed_tlist'),
+    @pytest.mark.parametrize(['form', 'base', 'kwargs'], [
+        pytest.param('cte', cte_qobj, {}, id='cte'),
+        pytest.param('func',
+                     [cte_qobj, [cplx_qobj, cplx_coeff.func]],
+                     {'args': args}, id='func'),
+        pytest.param('string',
+                     [cte_qobj, [cplx_qobj, cplx_coeff.string]],
+                     {'args': args}, id='string'),
+        pytest.param('spline',
+                     [[cplx_qobj, cplx_coeff.spline(tlist, args)], cte_qobj],
+                     {}, id='spline'),
+        pytest.param('array', [cte_qobj,
```

Comment:
If you prefer this spacing, at least be consistent.

Reply:
oh, that was a mistake. I didn't even mean to change the first bit

Reply:
or more, I didn't mean to commit it - I just fiddle around with it while I'm reading it

Reply:
I'll revert it and fix the commit